### DISCUSSION TITLE
[Feature] Image description based on CLIP embedding

### DISCUSSION THREAD

**Discussion by tux2000** [immich-app/immich#12900] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Ok, this probably only shows how little I understand the smart search / CLIP models used...

But would it be possible to do basically the opposite of smart search and use the text information in the CLIP model to generate an image description. The different concepts in the description could then also be links to start new smart searches.

An extension of this concept could be to show pictures with similar content base on the proximity in the CLIP feature space, i.e. image to image smart search...

This might be related to #2891 but I don't thinks its a duplicate

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2519, #4509
